DependencyInjection/DunglasActionExtension.php

```diff
@@ -108,6 +108,10 @@ private function getClasses($directory)
                 continue;
             }
 
+            if (version_compare(PHP_VERSION, '7.0.0') >= 0 && $reflectionClass->isAnonymous()) {
```

Comment:
I suggest to use feature detection instead:

`if (method_exists($reflectionClass, 'isAnonymous') && $reflectionClass->isAnonymous())`.